server/games/game.py

```diff
@@ -271,13 +272,15 @@ def add_game_connection(self, game_connection):
 
                 if self.desyncs > 20:
                     await self.mark_invalid(ValidityState.TOO_MANY_DESYNCS)
+                    return
 
                 if time.time() - self.launched_at > 4*60 and self.is_mutually_agreed_draw:
                     self._logger.info("Game is a mutual draw")
```

Comment:
Yes. Like many others. At least most have been fixed to be `debug` instead of `info`